mongo/options/findoptions.go

```diff
@@ -87,8 +89,10 @@ type FindOptions struct {
 	// The number of documents to skip before adding documents to the result. The default value is 0.
 	Skip *int64
 
-	// If true, the cursor will not return a document more than once because of an intervening write operation. This
-	// option has been deprecated in MongoDB version 4.0. The default value is false.
+	// If true, the cursor will not return a document more than once because of an intervening write operation. The
+	// default value is false.
+	//
+	// Deprecated: This option has been deprecated in MongoDB version 4.0.
```

Comment:
```suggestion
	// Deprecated: This option has been deprecated in MongoDB version 3.6 and removed in MongoDB version 4.0.
```

Reply:
According to https://jira.mongodb.org/browse/DRIVERS-454

Reply:
Changed for both `FindOptions` and `FindOneOptions`.